virttest/qemu_vm.py

```diff
@@ -3573,6 +3573,8 @@ def _get_pci_bus(self, params, dtype=None, pcie=False):
             else:
                 bridge_type = 'pci-bridge'
             return {'aobject': '%s-0' % bridge_type}
+        if machine_type.startswith('s390'):
```

Comment:
AFAIK, `css` is not a PCI bridge, so for `s390`, it should return `None` like `mmio`:
https://github.com/avocado-framework/avocado-vt/blob/6ab11ab8260d7888d092e00b0254b8c3ae25d420/virttest/qemu_vm.py#L3599-L3600

Reply:
I think it's different with "mmio", device "virtio-net-ccw" still need a bus, if returns None, could not get the right bus, I could change "pci_bus" to "css_bus" if you think "pci_bus" will cause confusion

Reply:
The function named `_get_pci_bus` so it should return a real PCI bus, for other bus types, need to handle them according to the actual situation.

Reply:
Modified, add new function named _get_css_bus